src/sagas/index.tsx

```diff
@@ -1,46 +1,52 @@
 import axios from "axios";
 import { takeEvery, call, all, put } from "redux-saga/effects";
-import { addTodoInStore } from "/home/mergestack/Desktop/Fakhar Training/React/React Hook Form/my-redux-hook-form-app/src/store/actions/index";
+import { addTodoInStore } from "/home/mergestack/Desktop/Fakhar Training/React/Form + Yup to Ts/my-redux-hook-form-ts-app/src/store/actions";
 
-function* createTodoSaga(action) {
+interface Action {
+  type: string;
+  payload: { _id: string; taskTitle: string };
+}
+interface addAction {
+  type: string;
+  payload: { taskTitle: string };
+}
+function* createTodoSaga(action: addAction): Generator {
   try {
-    const crudUrl = process.env.REACT_APP_CRUD_ENDPOINT_URL;
-    const response = yield call(axios.post, crudUrl, action.payload, {});
+    const crudUrl = process.env.REACT_APP_CRUD_ENDPOINT_URL || "";
+    const response: any = yield call(axios.post, crudUrl, action.payload);
     yield call(updateStore);
   } catch (error) {
     console.error(error);
   }
 }
 
-function* updateStore() {
+function* updateStore(): Generator {
   try {
-    const crudUrl = process.env.REACT_APP_CRUD_ENDPOINT_URL;
-    const response = yield call(axios.get, crudUrl);
+    const crudUrl = process.env.REACT_APP_CRUD_ENDPOINT_URL || "";
+    const response: any = yield call(axios.get, crudUrl);
     yield put(addTodoInStore(response.data));
```

Comment:
same any

Reply:
this one can't be removed because this will be sent to an action. 
I have tried its orignal type AxiosResponse with different variations, but it didn't work out. 
Tried unknown as well but didn't work out